internal/indexer/definitions/capybarabr.yaml

```
@@ -0,0 +1,136 @@
+---
+#id: CapybaraBR
+name: CapybaraBR
+identifier: capybarabr
+description: CapybaraBR (CBR) is a Brazilian general private torrent tracker focused on various types of content, including movies, TV shows, and more.
```

```suggestion
description: CapybaraBR (CBR) is a Brazilian general private torrent tracker focused on movies, TV shows, and more.
```
Keep it short and precise 😄